src/server/graphql.server.ts

```diff
@@ -67,6 +67,8 @@ export class PolarisGraphQLServer implements GraphQLServer {
             },
 
             formatResponse: (response: any) => {
+                if (response.data.__schema)
+                    return response;
```

Comment:
use curly brackets and check if the data starts with '__' (double underscore as const by the name of introspectionQueryPrefix)